packages/plugins/plugin-assistant/src/processor/processor.ts

```diff
@@ -72,93 +73,43 @@ export type AiRequest = {
  */
 export class AiChatProcessor {
   // TODO(burdon): Comment required.
+  // TODO(wittjosiah): Rename.
```

Comment:
perhaps just `_rx` (like `ctx`) would be better. "registry" implies something bigger (more relevant to the functionality) is going on -- or something directly connected to the observer (in terms of naming). i know that is their class name, but this is a very low-level thing we need to repeat?